packages/components/src/popover/index.js

```diff
@@ -42,405 +46,71 @@ const SLOT_NAME = 'Popover';
 
 const slotNameContext = createContext();
 
-function computeAnchorRect(
-	anchorRefFallback,
-	anchorRect,
-	getAnchorRect,
-	anchorRef = false,
-	container
-) {
-	if ( anchorRect ) {
-		return anchorRect;
-	}
-
-	if ( getAnchorRect ) {
-		if ( ! anchorRefFallback.current ) {
-			return;
-		}
-
-		const rect = getAnchorRect( anchorRefFallback.current );
-		return offsetIframe(
-			rect,
-			rect.ownerDocument || anchorRefFallback.current.ownerDocument,
-			container
-		);
-	}
-
-	if ( anchorRef !== false ) {
-		if (
-			! anchorRef ||
-			! window.Range ||
-			! window.Element ||
-			! window.DOMRect
-		) {
-			return;
-		}
-
-		// Duck-type to check if `anchorRef` is an instance of Range
-		// `anchorRef instanceof window.Range` checks will break across document boundaries
-		// such as in an iframe.
-		if ( typeof anchorRef?.cloneRange === 'function' ) {
-			return offsetIframe(
-				getRectangleFromRange( anchorRef ),
-				anchorRef.endContainer.ownerDocument,
-				container
-			);
-		}
+const positionToPlacement = ( position ) => {
+	const [ x, y, z ] = position.split( ' ' );
```

Comment:
What `z` could be? I thought Popover had only two parts like `top left` etc..

Reply:
It's a weird thing that Ella introduced. More details on this issue #21275 (it's still not entirely clear to me).